src/pudl_archiver/orchestrator.py

```diff
@@ -137,7 +137,7 @@ def __init__(
 
         self.create_new = create_new
         self.deposition_settings = deposition_settings
-        with open(deposition_settings) as f:
+        with Path.open(deposition_settings) as f:
             self.dataset_settings = {
                 name: DatasetSettings(**dois)
                 for name, dois in yaml.safe_load(f).items()
```

Comment:
I don't understand why, but switching to using `Path.open()` and then mocking that `Path.open()` in the tests seems to cause this call to `yaml.safe_load()` to get a string rather than a dictionary.